src/main/java/fruitshop/database/TransactionServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package fruitshop.database;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.model.TransactionService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionServiceImpl implements TransactionService {
+
+    @Override
+    public List<FruitTransaction> transactionData(List<String> readFile) {
+        List<FruitTransaction> transactionDao = new ArrayList<>();
+        String[] temp;
+        readFile.remove(0);
+        for (String line : readFile) {
+            temp = line.split(",");
+            transactionDao.add(FruitTransaction.of(temp[0],temp[1],Integer.parseInt(temp[2])));
+        }
+        return transactionDao;

```

Comment:
the variable transactionDao should be renamed, we're calling something dao when we're working with some sort of storage or DB, but here we're just implementing some logic that converts string into transaction, so let's just rename it to transactions

Reply:
thanks for your explaining